Update docs for credits release (#433)
ASFHyP3/hyp3-docs: docs/using/credits.md

# Credits

!!! warning "Attention"
    On **April 1st**, ASF's On Demand Monthly Processing Quota
    was replaced by a new **credits system**.
    Now, you are given an allotment of **10,000 credits per month**,

>     and each type of job will cost a different number of credits,

Reviewer: ```suggestion
    and each type of job costs a different number of credits,
```

    as shown in the table below.

>     Notably, you will now be able to run up to 10,000 Burst InSAR jobs per month—a 10x increase!

Reviewer: ```suggestion
    Notably, you can now run up to 10,000 Burst InSAR jobs per month—a 10x increase!
```
